ui/component/claimList/view.jsx

```diff
@@ -202,6 +204,7 @@ export default function ClaimList(props: Props) {
       dragHandleProps={draggableProvided && draggableProvided.dragHandleProps}
       unavailableUris={unavailableUris}
       showMemberBadge={showMemberBadge}
+      inHistory={inHistory}
```

Comment:
_Comment_
- This should probably not be that specific to "history".  It should be an optional "onClose" handler than we can pass from the History Page Component, which when `ClaimPreview` sees it, will display the "close" button. `ClaimPreview` should be as generic as possible.
  - But ok if want to leave as-is -- there is a ticket to fix other existing issues like this.